### PR TITLE
Auth - temp diagnosis page

### PR DESCRIPTION
**Pull request by psiinon**
A holding page which explains we'll be working more on the docs than answering questions.
Suggestions for improvements gratefully received :)

Screenshot 2022-04-27 at 12-18-28 OWASP ZAP – Diagnosing Authentication Problems

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: site/content/docs/authentication/diagnosing-auth-problems.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- guide
+type: page
+---
+
+If you ask a question related to authentication on one of the ZAP forums then you will be directed here.
+
+We know that the ZAP authentication documentation needs improving.
+One of the reasons why it has not been improved is that we are too busy trying to answer authentication questions :wink:.
+
+We are currently focusing on improving the authentication docs which means that will will not have as much time for answering specific questions.
```
Comment (kingthorin) [inline site/content/docs/authentication/diagnosing-auth-problems.md]:
```suggestion
We are currently focusing on improving the authentication docs which means that we will not have as much time for answering specific questions.
```

File: site/content/docs/authentication/diagnosing-auth-problems.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+This is why you have been redirected here :smile:.
+
+### Understand Your App
+
+Authentication is hard.
+We would love to be able to program ZAP to automatically understand all forms of authentication but that is a _long_ way off.
+
+We probably do not have access to your app, and even if we did we would not have the time to understand it for you.
+
+This means that you need to understand exactly have your app handles authentication (and session handling) in order to configure ZAP to handle it for you.
```
Comment (kingthorin) [inline site/content/docs/authentication/diagnosing-auth-problems.md:24]:
```suggestion
This means that you need to understand exactly how your app handles authentication (and session handling) in order to configure ZAP to handle it for you.
```

File: site/content/docs/authentication/diagnosing-auth-problems.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+### Existing Resources
+
+There are a lot of existing resources that can help you:
+
+* [ZAP Authentication](/docs/authentication/) - the docs we are currently working on
+* [Official ZAP Videos](/videos-list/) - search for "auth" in the "Tags" field
+* [User Group](https://groups.google.com/group/zaproxy-users) - search for "auth" or similar, more recent answers are likely to be more relevant
+* [Developer Group](https://groups.google.com/group/zaproxy-develop) - search for "auth" as per the User Group
```
Comment (kingthorin) [inline site/content/docs/authentication/diagnosing-auth-problems.md:33]:
```suggestion
* [Developer Group](https://groups.google.com/group/zaproxy-develop) - search for "auth" as per the User Group. (Please don't post your auth questions here, this group is for topics related to adding/changing ZAP code or add-ons. Though there are some historic threads that may assist you.)
```

**Comment by psiinon** [comment]
Thanks - updated.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**